### PR TITLE
Cloud compatibility

### PR DESCRIPTION
**Pull request by sophtank**
Implemented Gunicorn so we could push to the cloud. Also added some copy to the homepage. Homepage copy was supposed to be on the messages branch, I dunno what happened but I'm fine with it being added in this commit.

Information to connect to the cloud database is in the GroupMe. Once this pull request is approved, I will set the website to track #main instead of #cloud-push.

### CODE REVIEW

**Comment by skiser77** [comment]
This looks good to me my only concern is would a more up to date flask cause an issue? Probably not but just wondering. Everything else looks good. Home page looks solid.

**Comment by sophtank** [comment]
I don't think that a new flask version would cause an issue but we can keep an eye on it.

**Review by Zzz4nder — COMMENTED** [review]
You removed a lot of packages in requirements.txt. Are you certain those aren't dependencies?

The rest looks good!

File: requirements.txt [hunk 1]
```diff
@@ -1,22 +1,16 @@
 bcrypt==4.1.2
 blinker==1.7.0
 click==8.1.7
-colorama==0.4.6
-exceptiongroup==1.2.0
-Flask==3.0.2
```
[the remark this answers is not in the document]
Comment (Zzz4nder) [reply requirements.txt:6]: Theres more that I didn't highlight

**Pull request merged**